### ISSUE TITLE
Header for Standings report has redundant information when uploading reports to HSQB database

### ISSUE THREAD

**Issue by ani-per** [ANadig/YellowFruit#68]
When adding reports to the HSQB database, the information contained in the header for [the Standings report](https://hsquizbowl.org/db/tournaments/9054/stats/all_games/standings/) (tournament name, date, location, set name) is already in [the HSQB database entry](https://hsquizbowl.org/db/tournaments/9054/) and is displayed at the top of the page on each report (except for set name, but that's available at the main entry). I think this information is still valuable to contain in the .yft file, just stating that printing it in the header there may be unnecessary.

**Comment by jonahgreenthal** [comment]
I don't think YellowFruit's HTML output should assume that it's only for display on hsquizbowl.org; it should be useful when posted anywhere, and "useful" entails the inclusion of basic metadata like the tournament name. If anything, perhaps hsquizbowl.org should filter its uploads and/or modify its display so as to eliminate redundancy.

**Comment by ani-per** [comment]
> I don't think YellowFruit's HTML output should assume that it's only for display on hsquizbowl.org; it should be useful when posted anywhere, and "useful" entails the inclusion of basic metadata like the tournament name. If anything, perhaps hsquizbowl.org should filter its uploads and/or modify its display so as to eliminate redundancy.

Yes, I agree fully. Just wanted to bring this to Andrew's attention in case he considers HSQB to be the primary use case. This could also be a configurable option with printing being the default.

**Comment by ANadig** [comment]
Yes, Jonah's reasoning is exactly my logic here. It's worth noting that those fields are entirely optional, and nothing bad happens whatsoever if you leave them blank (unless I guess you're sending the file to some other tool that needs them).

And for the question set name in particular, I considered it a flaw that you had to navigate back out of the stat report to see it.

**Comment by ANadig** [comment]
Closing as not planned, since the workaround of leaving those fields blank is very easy.
